sway/commands/input/xkb_file.c

```diff
@@ -0,0 +1,22 @@
+#define _POSIX_C_SOURCE 200809L
+#include "sway/config.h"
+#include "sway/commands.h"
+#include "sway/input/input-manager.h"
```

Comment:
Also, this include isn't needed

Reply:
Again, this file is almost an exact copy and paste from xkb_layout.c. Should these all be changed?

Reply:
Yes, it's not needed in any of them. It's probably a remnant of a previous iteration.